src/Microsoft.AspNet.Mvc.Core/ApplicationModels/ActionModel.cs

```diff
@@ -76,5 +78,14 @@ public ActionModel([NotNull] ActionModel other)
         public IList<ParameterModel> Parameters { get; private set; }
 
         public IList<IRouteConstraintProvider> RouteConstraints { get; private set; }
+
+        /// <summary>
+        /// Stores arbitrary metadata properties associated with the action.
```

Comment:
The comment doesn't explain what this will do.

It should say something like - Gets a set of properties that will be copied to the ActionDescriptor.Properties